CHANGELOG.rst

```diff
@@ -21,7 +21,7 @@ v3.0.0
 * Make the output of move, delete, copy and flush more consistent with
   everything else.
 * Porcelain now outputs proper JSON, rather than one-JSON-per-line.
-
+* todo does not output anything when their are no list.
```

Comment:
Please rewrite this. This changelog should explain in simple term to users what changed. This explains what the bug was, not what we changed.